controllers/acquire_cube.py

```diff
@@ -48,6 +48,13 @@ def waiting_for_cube(self) -> None:
             self.intake.deploy_without_running()
             self.next_state("grabbing")
```

Comment:
@james-ward why was this changed back to skip the ```clamping``` state? Did we change our mind after I left on Monday? If we don't want this anymore then the whole clamping state can be removed

Reply:
Clamping cannot work unless the intake is moved forwards. I left the unused state there in case we want it in the future.

Reply:
Cannot work? What do you mean by that? does it put holes in the cube or something?

Reply:
You can't pull the cube up with the intake retracted.